System.IO.Abstractions.TestingHelpers/MockFileInfo.cs

```diff
@@ -7,11 +7,14 @@ public class MockFileInfo : FileInfoBase
     {
         private readonly IMockFileDataAccessor mockFileSystem;
         private string path;
+        private string originalPath;
 
         public MockFileInfo(IMockFileDataAccessor mockFileSystem, string path) : base(mockFileSystem?.FileSystem)
         {
             this.mockFileSystem = mockFileSystem ?? throw new ArgumentNullException(nameof(mockFileSystem));
-            this.path = path ?? throw new ArgumentNullException(nameof(path));
+            this.originalPath = path ?? throw new ArgumentNullException(nameof(path));
+            this.path = mockFileSystem.FixPath(path);
```

Comment:
Can this just call `mockFileSystem.Path.GetFullPath(path)`?

Reply:
indeed, fixed!